modules/core/src/main/scala/org/scalasteward/core/forge/ForgeType.scala

```diff
@@ -38,35 +42,64 @@ sealed trait ForgeType extends Product with Serializable {
 }
 
 object ForgeType {
+  trait DiffUriPattern { def forDiff(from: String, to: String): Uri => Uri }
+  trait FileUriPattern { def forFile(fileName: String): Uri => Uri }
+
   case object AzureRepos extends ForgeType {
     override val publicWebHost: Some[String] = Some("dev.azure.com")
     override def supportsForking: Boolean = false
+    val diffs: DiffUriPattern = (from, to) =>
+      _ / "branchCompare" withQueryParams Map(
+        "baseVersion" -> s"GT$from",
+        "targetVersion" -> s"GT$to"
```

Comment:
**suggestion:** You can also use the operator to add query params `+?`

```scala
_ / "branchCompare" +? ("baseVersion", s"GT$from") +? ("targetVersion", s"GT$to")
```

<sub>This comment follows the [**conventionalcomments.org**](https://conventionalcomments.org) standard</sub>

Reply:
Ooh, nice one! Updated with fca39d49657e0a4dd7e09f73b642b6654b947f56 👍